lib/ukschedcoop/include/uk/schedcoop.h

```diff
@@ -44,7 +44,10 @@
 extern "C" {
 #endif
 
-struct uk_sched *uk_schedcoop_create(struct uk_alloc *a);
+struct uk_sched *uk_schedcoop_create(struct uk_alloc *a,
```

Comment:
Now that we have several allocators I would be cleaner if we passed a struct for allocators similarly to `ops`.

Reply:
For now, the creation functions for scheduler initialization are custom and defined by the scheduler implementation, likewise the allocator initialization. There is no common init interface layout defined yet for schedulers and allocators. I am also not sure if we could define one because schedulers and allocators could handle very specific cases.

Reply:
Concluding: I think it is okay to expand this definition with this PR. The only place where the scheduler is initialized is in ukboot atm, impact should be minimal for this particular case.